baseballAPI.UnitTests/Controllers/PlayerControllerIntegrationTests.cs

```diff
@@ -41,22 +41,15 @@ public void SetUp()
             }).CreateClient();
         }
 
-        [TestCase("/Player", "first", "last", "something")]
-        [TestCase("/Player", "Sammy", "Sosa", "Ssosa")]
-        public async Task GetPlayerIdByFirstAndLastName(string url, string firstName, string lastName, string expectedPlayerId)
+        [Test]
+        public async Task GetPlayerIdByFirstAndLastName()
         {
-            _playerService.Setup(mockPlayerService => mockPlayerService.GetPlayerId(firstName, lastName)).Returns(expectedPlayerId);
+            var firstPerson = GeneratePeople("first", "last", "something");
+            var secondPerson = GeneratePeople("Sammy", "Sosa", "Ssosa");
+            var url = "/player";
 
-            var uri = QueryHelpers.AddQueryString(url, new Dictionary<string, string> {
-                {"firstName", firstName},
-                {"lastName", lastName}
-            });
-
-            var response = await _client.GetAsync(uri);
-
-            response.EnsureSuccessStatusCode();
-            var content = await response.Content.ReadAsStringAsync();
-            Assert.That(content, Is.EqualTo(expectedPlayerId));
+            //await AssertThatClientResponseEqualsExpectedResponse(url, firstPerson);
```

Comment:
probably didn't mean to leave these commented out.
I imagine these are cases we care about proving.